src/containers/app/index.js

```diff
@@ -119,22 +129,22 @@ class App extends React.Component {
         return (
             <div className={showConnection ? "Site App" : "Site"}>
                 <ScrollToTop />
-                {showConnection &&
-                    <DisclaimerModal />
-                }
-                <TopNav web3Connect={this.props.web3Connect} className={!showConnection && "hide"} />
-                {!showConnection &&
+                {showConnection && <DisclaimerModal />}
+                <TopNav
+                    web3Connect={this.props.web3Connect}
+                    toggleAccInfo={this.toggleAccInfo}
+                    showAccInfo={this.state.showAccInfo}
+                    className={!showConnection && "hide"}
+                />
+                {!showConnection && (
                     <SiteMenu
                         web3Connect={this.props.web3Connect}
                         location={this.props.location}
                         showMenu={this.state.showMobileMenu}
                         toggleMenu={this.toggleMenu}
                     />
-                }
-                {showConnection 
-                    ? <SideNav showMenu={this.state.showMobileMenu} toggleMenu={this.toggleMenu} /> 
-                    : null
-                }
+                )}
+                {showConnection ? <SideNav showMenu={this.state.showMobileMenu} toggleMenu={this.toggleMenu} /> : null}
```

Comment:
please do not use ternary operator if it is not necessary.

`{showConnection && <SideNav showMenu={this.state.showMobileMenu} toggleMenu={this.toggleMenu} />}`